recipes-security/refpolicy/files/customizable_types

```diff
@@ -0,0 +1 @@
+user_tty_device_t
```

Comment:
commit message doesn't match changes

Reply:
I've changed the commit and made it more informative. Is it ok, or should I add something else?